# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1368407119 14400
#      Sun May 12 21:05:19 2013 -0400
# Node ID 4258750c76eddedcd29e2f9a0f88c8db8a3fa688
# Parent  e3a0ca9c883627c03e53a713d533ab9dcb5f77f5
don't push operator tokens twice

* lex.ll (octave_base_lexer::handle_op_internal):
Use count_token_internal, not count_token.

diff --git a/libinterp/parse-tree/lex.ll b/libinterp/parse-tree/lex.ll
--- a/libinterp/parse-tree/lex.ll
+++ b/libinterp/parse-tree/lex.ll
@@ -3029,17 +3029,17 @@ octave_base_lexer::handle_op_internal (i
     gripe_matlab_incompatible_operator (flex_yytext ());
 
   push_token (new token (tok, input_line_number, current_input_column));
 
   current_input_column += flex_yyleng ();
   looking_for_object_index = false;
   at_beginning_of_statement = bos;
 
-  return count_token (tok);
+  return count_token_internal (tok);
 }
 
 int
 octave_base_lexer::handle_token (const std::string& name, int tok)
 {
   token *tok_val = new token (tok, name, input_line_number,
                               current_input_column);
 
